### DISCUSSION TITLE
CodeSandbox preview not loading due to domain restriction

### DISCUSSION THREAD

**Discussion by arunkumaralagesan** [codesandbox/codesandbox-client#8307] [Q&A]
I have been trying to use the preview panel to test my app, but the preview is failing to load.
But i remember few weeks back i updated "csb.app" to "codesandbox.io" and it worked. But now that is also not working.

<img width="1725" alt="Screenshot 2024-01-30 at 9 06 19 PM" src="https://github.com/codesandbox/codesandbox-client/assets/50190026/c4ce25d3-08c9-42af-a068-aeff68f71f2d">

please refer the below link for reference [sandbox](https://codesandbox.io/p/sandbox/gridstack-js-o03d4?file=%2Fsrc%2FNestedGrid.jsx)

Thanks!

**Comment by CompuIves** [comment]
Hello! Could it be that a corporate proxy is blocking `csb.app`? We recently started to redirect `xxx.codesandbox.io` to `xxx.csb.app` because some entities (like Microsoft) started marking codesandbox.io as dangerous because some phishing sandboxes were hosted on `codesandbox.io`.

**Reply by arunkumaralagesan** [reply]
Thanks for the reply!, and yes, I am trying this in corporate laptop and my company falls under the Microsoft corporation. Is there any workaround for it? TIA!
